### DISCUSSION TITLE
Notifications subscriptions for business

### DISCUSSION THREAD

**Discussion by matiaslopezd** [mdn/mdn-community#154] [MDN Plus] [answered]
Hi guys! I love MDN plus! For now, I'm using the developer subscription, payed version is not available in my country yet.

Now I'm receiving browser compatibility updates through notifications, which is cool, saving a lot of time testing on devices based on the tables like with WebRTC.

But I think it could be very profitable for MDN and valuable for all companies to receive updates of the compatibility tables via webhooks. For example, we could create Github issues when compatibility is added or deprecated. Inclusive could allow knowing the browser compatibility automatically based on your code using an updated webhooks service.

Think about the millions of companies that need to know if a feature is available for browsers but updated. MDN today is a source of truth for millions of developers.

What do you think?

**Comment by Rumyra** [comment, marked as answer]
Thanks for your suggestion @matiaslopezd :) The plus team are taking it into consideration 👍
